test/test_helper.rb

```diff
@@ -148,6 +150,15 @@ def unauthorized(method, action, params = {})
         end
       end
     end
+
+    %w{deployer_project_admin}.each do |user|
```

Comment:
It can. I only kept it separated because the first group of Users represent system-level roles, and this second group are meant for testing Users with project-specific roles.

I'll refactor it.